extensions/shared/with-custom-class-names/style.scss

```diff
@@ -0,0 +1,13 @@
+// Override core styles inherited from `.has-warning`:
+// we do want blocks with upgrade nudge warning to be interactive
+.block-editor-block-list__layout .block-editor-block-list__block.has-warning.is-interactive,
```

Comment:
This doesn't seem great. What is the reasoning for making the block interactive?

Reply:
Not sure was thinking the same yesterday. I'm just changed/updated this because I changed of filter to apply the CSS class.